### DISCUSSION TITLE
distriqt ANE News Update (May 30, 2024)

### DISCUSSION THREAD

**Discussion by 2jfw** [airsdk/Adobe-Runtime-Support#3284] [General]
Just came across a recent nice-to-read news item from distriqt I thought would be worth to share, as it may be missed or be unnoted:

https://docs.airnativeextensions.com/news/2024-05

Since there aren't many websites left where AIR/as3 related news can be found I would think that here would be a suitable place for something like this (maybe also have a "News / Announcement" category for discussion threads?).

Also, thx @marchbold for your passion on this, very much appreciated!

**Comment by ajwfrost** [comment]
Sounds good - thanks @marchbold for that :-)

This sort of thing can probably be added to the "announcements" discussion item. It doesn't have to just be about new SDK releases :-) 
But yes, also a "category" item would be useful for this sort of thing, rather than just "General" ..

**Comment by raresn** [comment]
@marchbold thank you for all your hard work! Same goes for @ajwfrost and your team, of course!
Can't wait to see whats next!

**Comment by marchbold** [comment]
Hi all, thanks for the feedback! Must say I was feeling a little guilty about not having the time to update our news feed a bit more regularly, but I'm hoping to get back into a monthly (or at least bimonthly) update for the remainder of the year. 

Cheers
